lib/mix/lib/mix/release.ex

```diff
@@ -237,11 +237,24 @@ defmodule Mix.Release do
   end
 
   defp load_app(app, seen, otp_root) do
+    path = Path.join([otp_root, "#{app}-*"])
+
+    case Path.wildcard(path) do
+      [] ->
     case :code.lib_dir(app) do
```

Comment:
We probably don't need this branch anymore, we can fully rely on `Path.wildcard`.

Reply:
Actually, we do need this branch, for non OTP apps. So yeah, we just need to format and we should be good to go.